### DISCUSSION TITLE
Num-Enter ignored

### DISCUSSION THREAD

**Discussion by davanderson** [Extended-Thunder/send-later#526] [Q&A]
Is there something I should be doing for Send Later to recognize the numeric keypad enter key for completing the send operation on messages?  I've looked into keymap changes but haven't found a solution.  This is working on my HP laptop but not on my desktop.

Background ... Debian 11 using Thunderbird v102.8.0 (64-bit).  Logitech Wave keyboard (WiFi)

**Comment by jikamens** [comment]
Tracking as issue in #567.

**Comment by jikamens** [comment]
I believe this is resolved in release 10.0.2 of the add-on. If it isn't, please let me know.
